mobileapp/components/game/Game.tsx

```diff
@@ -36,6 +44,20 @@ export const Game = () => {
     fetchGameData();
   }, []);
 
+  const play = ({ x, y }: TileCoordinate) => {
```

Comment:
Only Tiles have to use this function and they could do it from a context instead of relying on BoardRenderer.
Maybe we need a BoardContext ?

Reply:
We could do this when the component will growth and when he will needs more functions ? Are you ok with that ?